MapboxNavigation.xcodeproj/project.pbxproj

```diff
@@ -1977,6 +1971,7 @@
 			developmentRegion = English;
 			hasScannedForEncodings = 0;
 			knownRegions = (
+				English,
```

Comment:
Xcode 10.2 replaced `English` with `en`. I wonder if this can cause any issues. 🤔

Reply:
Xcode 10.2 adds this entry automatically upon opening the project. I’m not sure why, given that `en` is already in the list and we don’t have any English.lproj folders in the repository. But the project functions fine with or without this change.

Reply:
Huh, today Xcode 10.2 warned me about this entry and offered to remove it. I guess it was having a different mood today. Removed in a016c58a0e7bc4d7d3d53aeaca2a09f474a92840.